quadra/tasks/classification.py

```diff
@@ -981,29 +962,9 @@ def deployment_model(self, model_path: str):
 
     def prepare(self) -> None:
         """Prepare the evaluation."""
-        with open(os.path.join(Path(self.model_path).parent, self.deploy_info_file)) as f:
-            self.model_data = json.load(f)
-
-        if not isinstance(self.model_data, dict):
-            raise ValueError("Model info file is not a valid json")
-
-        if self.model_data["input_size"][0] != self.config.transforms.input_height:
-            log.warning(
-                f"Input height of the model ({self.model_data['input_size'][0]}) is different from the one specified "
-                + f"in the config ({self.config.transforms.input_height}). Fixing the config."
-            )
-            self.config.transforms.input_height = self.model_data["input_size"][0]
-
-        if self.model_data["input_size"][1] != self.config.transforms.input_width:
-            log.warning(
-                f"Input width of the model ({self.model_data['input_size'][1]}) is different from the one specified "
-                + f"in the config ({self.config.transforms.input_width}). Fixing the config."
-            )
-            self.config.transforms.input_width = self.model_data["input_size"][1]
-
-        self.deployment_model = self.model_path
-
         super().prepare()
+        self.deployment_model = self.model_path
```

Comment:
Why this is not done in the super? We need to manually assign everytime or I'm missing something

Reply:
Ok I saw it may be different for plain torch import

Reply:
Actually the only "problem" exists for patchsklearn and sklearnclassif. which don't have a deployment model but backbone + classifier. To move the self.deployment_model = self.model_path in Evaluation i will overwrite the deployment model setter and set classifier and backbone in there. Without really seting a deployment_model object, which is not used in patchsklearn and sklearnclassification.

Reply:
Done, see if you like that. Commit: style: Move self.deployment_model = self.model_path to parent class
p.s. i know it's not "style", "refactor" would have been better